CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
Can you completely remove any changes to this file? I'll update the changelog when I cut a new release.